src/components/sections/ProductGallery/product-gallery.scss

```diff
@@ -85,8 +58,11 @@
 
 .product-listing__results {
   --padding: var(--space-1);
+  --product-listing-grid-height: 2197px;
```

Comment:
I guess this is also temporary right?

Reply:
I got this number testing in different screen sizes and it is the minimum when we have the page loaded with the products. If we ensure that the content that will fill in the loading space (skeletons?) has at least this height we can remove it afterward. this height ensures good CLS so far.

Reply:
Oh great! How about a ticket on Jira for follow-up/check. Or a comment here? //Check it after apply Skeleton.

Reply:
Nice, Done it here 9a7948dbb2f8b9828ef146d7e49e5b9185e9b8e9